public/js/chunk-3b778467.04153c2b.js

```diff
@@ -1,2 +1,2 @@
ers.push(n);else if("remove"===n.event&&"folder"===n.type){for(var i=-1,r=0;r<t.folder.folders.length;r++)if(t.folder.folders[r].path===n.path){i=r;break}i>-1&&t.folder.folders.splice(i,1)}else if("remove"===n.event&&"file"===n.type){for(var a=-1,s=0;s<t.folder.files.length;s++)if(t.folder.files[s].path===n.path){a=s;break}a>-1&&t.folder.files.splice(a,1),a=-1;for(var o=0;o<t.selectedFiles.length;o++)if(t.selectedFiles[o].path===n.path){a=o;break}a>-1&&t.selectedFiles.splice(a,1),a=-1;for(var l=0;l<t.screenFiles.length;l++)if(t.screenFiles[l].path===n.path){a=l;break}a>-1&&t.screenFiles.splice(a,1)}}},setFocusOnFiles:function(){window.document.getElementById("keyupevents").focus()},enablePreventingScrolling:function(){window.addEventListener("keydown",pe,!1)},disablePreventingScrolling:function(){window.removeEventListener("keydown",pe)},onOpenModal:function(){this.disablePreventingScrolling()},onCloseModal:function(){this.setFocusOnFiles(),this.enablePreventingScrolling()},showStatistic:function(){b["a"].$emit("show-statistic",this.path)},createNewFolder:function(){console.log("create new folder"),b["a"].$emit("create-new-folder",this.path)},shiftingMove:function(e,t){e&&e.shiftKey&&this.cursor(t,!0)}},created:function(){var e=this;return Object(l["a"])(regeneratorRuntime.mark((function t(){var n,i;return regeneratorRuntime.wrap((function(t){while(1)switch(t.prev=t.next){case 0:return e.perPage=e.configFilePerPage,t.next=3,e.loadFiles(e.dir||null);case 3:return n=t.sent,i=n.currentPath,e.createdFolders=e.folder.folders,t.next=8,e.loadStatistic(e.dir||i||e.path||null);case 8:return e.path=i,e.openedPath=i,f["a"].subscibeForFolder(e.path,e.fileChanged()),e.setFocusOnFiles(),t.next=14,h["a"].getRunningState();case 14:e.status=t.sent,f["a"].subscibeForFolder("running.lock",(function(t){"change"===t.event&&(e.status=t.content),"unlink"===t.event&&(e.status=!1)})),e.enablePreventingScrolling(),e.$store.commit("notes/SET_FOLDER",{notesPath:"".concat(i,"/notes.txt"),notes:e.systemConfig.notes||null,highlight:Boolean(e.systemConfig.highlight)});case 18:case"end":return t.stop()}}),t)})))()},beforeDestroy:function(){f["a"].unsubscribeForFolder(this.path),f["a"].unsubscribeForFolder("running.lock")}},_e=me,ve=(n("0117"),Object(y["a"])(_e,a,s,!1,null,null,null)),xe=ve.exports,ye={props:["dir","type"],components:{FileExplorer:xe},methods:{checkPermission:function(){}},mounted:function(){b["a"].$on("loaded-config",this.checkPermission)},destroyed:function(){b["a"].$off("loaded-config")}},we=ye,ke=Object(y["a"])(we,i,r,!1,null,null,null);t["default"]=ke.exports},c740:function(e,t,n){"use strict";var i=n("23e7"),r=n("b727").findIndex,a=n("44d2"),s=n("ae40"),o="findIndex",l=!0,c=s(o);o in[]&&Array(1)[o]((function(){l=!1})),i({target:"Array",proto:!0,forced:l||!c},{findIndex:function(e){return r(this,e,arguments.length>1?arguments[1]:void 0)}}),a(o)},e03a:function(e,t,n){},f748:function(e,t){e.exports=Math.sign||function(e){return 0==(e=+e)||e!=e?e:e<0?-1:1}}}]);
```

Comment:
@petrovfedor Make the change https://github.com/com2u/SVTrain/blob/4a84d65d294eb3f158b86edbcde59bb4558fa85a/client/src/components/ShowImage.vue#L155 and rebuild the bundle to make it preerved

Reply:
Haha. So the uncompiled javascript was there all the time? Good.